data/weapons/weapon_data.csv

```diff
@@ -99,14 +99,14 @@ Deep Strike Pod,diableavionics_station_deepStrike,4,,350,10000,,500,2000,500,0,5
 #$Harvest (semi-auto),diableavionics_versant_harvestC_LEFT,4,,,850,,150,,,180,0,6,1.25,3,ENERGY,200,,0,0.05,9999,0.5,0,15,2.5,2.5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,Strike,,,,,,,%s: can fire as fast as the triger is pulled.,Semi-auto weapon,,
 #Harvest (semi-auto),diableavionics_versant_harvestC_RIGHT,4,,,850,,150,,,180,0,6,1.25,3,ENERGY,200,,0,0.05,9999,0.5,0,15,2.5,2.5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,Strike,,,,,,,%s: can fire as fast as the triger is pulled.,Semi-auto weapon,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Harvest (auto),diableavionics_versant_harvest_LEFT,4,,,750,,100,,,180,0,,,,ENERGY,115,,0,0.25,,,0,20,1,2.5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,Assault,,,,,,,,,,
-Harvest (auto),diableavionics_versant_harvest_RIGHT,4,,,750,,100,,,180,0,,,,ENERGY,115,,0,0.25,,,0,20,1,2.5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,Assault,,,,,,,,,,
+Harvest (auto),diableavionics_versant_harvest_LEFT,4,,,700,,100,100,,180,0,,,,ENERGY,90,,0,0.25,,,0,20,1,2.5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,Assault,,,,,,,,,,
+Harvest (auto),diableavionics_versant_harvest_RIGHT,4,,,700,,100,100,,180,0,,,,ENERGY,90,,0,0.25,,,0,20,1,2.5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,Assault,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Harvest (burst),diableavionics_versant_harvestB_LEFT,4,,,800,,100,,,180,0,,,,ENERGY,100,,0,0.8,3,0.1,0,5,1,5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,General,,,,,,,,,,
-Harvest (burst),diableavionics_versant_harvestB_RIGHT,4,,,800,,100,,,180,0,,,,ENERGY,100,,0,0.8,3,0.1,0,5,1,5,,2000,,,100,,,SYSTEM,"no_drop, no_sell, no_drop_salvage",,Diable Avionics,,General,,,,,,,,,,
+Harvest (burst),diableavionics_versant_harvestB_LEFT,4,,,800,,150,,,180,0,,,,ENERGY,70,,0,0.8,3,0.1,0,5,1,5,,2000,,,100,,,"ANTI_FTR,PD_ALSO,SYSTEM","no_drop, no_sell, no_drop_salvage",,Diable Avionics,,General,,,,,,,,,,
+Harvest (burst),diableavionics_versant_harvestB_RIGHT,4,,,800,,150,,,180,0,,,,ENERGY,70,,0,0.8,3,0.1,0,5,1,5,,2000,,,100,,,"ANTI_FTR,PD_ALSO,SYSTEM","no_drop, no_sell, no_drop_salvage",,Diable Avionics,,General,,,,,,,,,,
```

Comment:
Harvest (burst) damage set to 150. Flux cost reduced to 70 (from 100). will target fighters if autofiring.